File: blind-JS45/binary/4-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n = len(nums)
        result = 0
        for i in range(1, n + 1):
            result ^= i
        for num in nums:
            result ^= num
        return result

File: blind-JS45/binary/4-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int result = 0;
        for (int i = 1; i <= n; i++) {
            result = result ^ i;
        }
        for (int i = 0; i < n; i++){
            result = result ^ nums[i];
        }
        return result;
    }
}